src/pages/media.js

```diff
@@ -1,9 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Layout from '../components/dev-hub/layout';
+import Tab from '../components/dev-hub/tab';
 import useAllVideos from '../hooks/use-all-videos';
-import { P } from '../components/dev-hub/text';
-import CardList from '../components/dev-hub/card-list';
 import usePodcasts from '../hooks/use-podcasts';
+import { P } from '../components/dev-hub/text';
+import CardList from '../components/dev-hub//card-list';
```

Comment:
```suggestion
import CardList from '../components/dev-hub/card-list';
```

Looks like an extra / here